packages/geolocation/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@booking-services/geolocation",
+  "version": "1.0.0",
+  "description": "This package contains all business rules for geolocation service",
+  "author": "Booking Backend Team",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/master/packages/geolocation#readme",
+  "license": "ISC",
+  "main": "src/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
+    "src": "src"
+  },
+  "files": [
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Edilberto-Vazquez/booking-services.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/Edilberto-Vazquez/booking-services/issues"
+  },
```

Comment:
@Edilberto-Vazquez can we change these URLs? because the URLs should be the main URLs and not the fork URLs

Reply:
I change the URLs to the main repository